graylog2-server/src/main/java/org/graylog/security/authservice/backend/LDAPAuthServiceBackend.java

```diff
@@ -77,23 +77,24 @@ public LDAPAuthServiceBackend(UnboundLDAPConnector ldapConnector,
 
             final LDAPUser userEntry = optionalUser.get();
 
-            if (!authCredentials.isAuthenticated()) {
-                if (!isAuthenticated(connection, userEntry, authCredentials)) {
-                    LOG.debug("Invalid credentials for user <{}> (DN: {})", authCredentials.username(), userEntry.dn());
-                    return Optional.empty();
-                }
-            }
-
             final UserDetails userDetails = provisionerService.provision(provisionerService.newDetails(this)
                     .authServiceType(backendType())
                     .authServiceId(backendId())
+                    .accountIsEnabled(true)
                     .base64AuthServiceUid(userEntry.base64UniqueId())
                     .username(userEntry.username())
                     .fullName(userEntry.fullName())
                     .email(userEntry.email())
                     .defaultRoles(backend.defaultRoles())
                     .build());
 
+            if (!authCredentials.isAuthenticated()) {
```

Comment:
Can you please add a comment about why we first provision and then authenticate?

Reply:
I've changed my mind about changing this. We'll have to refactor this at some point anyhow.
I'm leaving this untouched, as this is too close to release